manual/installation.tex

```diff
@@ -392,10 +422,11 @@ \subsection{Using vendor optimized numerical libraries, e.g. Intel MKL}
 \subsubsection{Using Intel MKL with non-Intel compilers}
 
 To use Intel MKL with, e.g. an MPICH wrapped gcc:
+\verbatimfont{\footnotesize}
 \begin{verbatim}
 cmake \
  -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx \
- -DBLA_VENDOR=Intel10_64lp_seq -DCMAKE_PREFIX_PATH=$MKLROOT/lib \
+ -DBLA_VENDOR=Intel10_64lp_seq -DENABLE_MKL=1 -DMKL_ROOT=$MKLROOT/lib \
```

Comment:
I think if ENABLE_MKL=1 is enabled, BLA_VENDOR is skipped completely.

Reply:
Correct should be deleted